status.go

```diff
@@ -201,8 +203,10 @@ func (s *sequins) serveStatus(w http.ResponseWriter, r *http.Request) {
 		}
 
 		for _, db := range status.DBs {
-			for versionName, version := range db.Versions {
-				db.Versions[versionName] = calculateReplicationStats(version)
+			for versionName := range db.Versions {
+				vst := db.Versions[versionName]
+				vst.calculateReplicationStats()
+				db.Versions[versionName] = vst
```

Comment:
Why do we need this assignment?

Reply:
@bartle-stripe  I can't modify it in place because apparently you can't take the address of map values in go. The alternative was to make the function not take a receiver and just reassign it, but that didn't really seem any cleaner.

Reply:
I guess you could change `db.Versions` to `map[string]*versionStatus`, but meh.